### ISSUE TITLE
Catch pokemon and dissapear from the map

### ISSUE THREAD

**Issue by NADDYALE** [daydreamer1222/WeCatch-Android#19]
If I use the pokemon account email address  (gmail) the pokemon dissapear from the más when capturing it?

**Comment by daydreamer1222** [comment]
this app does not interact with any of your Pokemon Go account

**Comment by NADDYALE** [comment]
Thank you for answering me... What are the benefits of register so? BTW, I updated it with the latest version published here at github, and it is not working. It says it conflicting with the Google services... Any idea?

**Comment by aidrouge** [comment]
Yes yes problem with play service for Android spoofer downgrading google service

**Comment by NADDYALE** [comment]
It is 12.6.85 🤷‍♂️

**Comment by aidrouge** [comment]
Damn wheres older releases 😣😤
